### ISSUE TITLE
Improve User creation.

### ISSUE THREAD

**Issue by richardkmichael** [richardkmichael/buy-my-beer#20]
When a user logs in using their GitHub credentials, we create a stub user account.

Should we track this at all?  We need it for collaborations, so we'd have to re-work collaborations as well.

If we completely depend on GitHub only, we could leverage their projects, collabs, etc..

app/models/user.rb
- A trigger here, to alert the user we've created an account for them.
- Fire an email if there's an address?  Display a notification in the view?  What about API?
- If the user has no email, this will fail.

**Comment by franckverrot** [comment]
I dont think you really need to tell him you created an user in your system. The app just uses its local storage as a proxy to the GH API.
